db/schema.rb

```diff
@@ -0,0 +1,66 @@
+# rubocop:disable all
```

Comment:
- Hi, Can you erase this commented code to avoid confusion? The requirement prohibits disabling the linter. [Rubocop](https://docs.rubocop.org/en/stable/) is a Ruby static code analyzer (a.k.a. linter) and code formatter. It will enforce many of the guidelines outlined in the community [Ruby Style Guide](https://rubystyle.guide/).